### ISSUE TITLE
Make a deposit

### ISSUE THREAD

**Issue by jemymaRC** [jemymaRC/KasaNaTua#1]
Customer dials 721
Welcome to Dalex Swift.  
1.	To make a deposit press 1
2.	To check your balance press 2
3.	To change your pin press 3
4.	To hear this menu again 4

To make a deposit press 1
1.1 To make a deposit for self Press 1
1.2 To make a deposit for a third party Press 2

Press 1.1
a.	Check database for customer number:
1.	If number OK: GOTO b.
2.	If number NOT OK: this number does not exist. Please hold to speak to our call center agents for assistance

b.	Please enter the amount to deposit on your key pad and press # to confirm
Press# :
IVR: the amount to deposit is XX.XX Ghana Cedis: 
IVR: Press 1 to accept or press 2 to enter amount again:
i.	Press 1 to accept amount
1.	if amount OK: GOTO c.
ii.	Press 2 to enter amount again
1.	if amount OK: GOTO c.
2.	If amount NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent

c.	Please enter your pin and # to confirm
i.	If pin OK: IVR: your request is now being processed. Thank You: GOTO c.
ii.	If pin NOT OK: that is the wrong pin please try again
iii.	If pin NOT OK after 3 attempts: sorry you having problems with your pin please wait while we connect you to our call center agent.
d.	Please press 1 to return to the main menu else hang up. Thank you

Press 1.2
a.	Please enter the customer’s mobile number and press # to confirm
Press# :
IVR: the number you entered is XXXXXXXXXX: 
IVR: Press 1 to accept or press 2 to enter number again:
i.	Press 1 to accept number:
1.	If number OK: GOTO b.
2.	If number NOT OK: the number does not exist please try again (Number not in DB)
3.	If number NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent

ii.	Press 2 to enter number again:
1.	If number OK: GOTO b.
2.	If number NOT OK: the number does not exist please try again
3.	If number NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent

b.	Please enter the amount to deposit on your key pad and press # to confirm
Press# :
IVR: the amount to deposit is XX.XX Ghana Cedis: 
IVR: Press 1 to accept or press 2 to enter amount again:
i.	Press 1 to accept amount
1.	if amount OK: GOTO b.
ii.	Press 2 to enter amount again
1.	if amount OK: GOTO b.
2.	If amount NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent
b.	Please enter your pin and press # to confirm
i.	If pin OK: your request is now being processed. Thank You.: GOTO d.
ii.	If pin NOT OK: that is the wrong pin please try again
iii.	If pin NOT OK after 3 attempts: sorry you having problems with your pin please wait while we connect you to our call center agent.
c.	Please press 1 to return to the main menu else hang up. Thank you.

**Comment by jemymaRC** [comment]
@franklinodoi Please test this functionality/menu option thoroughly using the step-by-step commented above and add the label "Passed Functional Test" if it passes or "Failed Test" if it failed (with details as to what failed.

**Comment by jemymaRC** [comment]
@franklinodoi I've reassigned this issue to myself. So you nolonger need to test this.

**Comment by jemymaRC** [comment]
Making a deposit for self:

Step 1: To make a deposit Press 1 - WORKING
Step 2: To make a deposit for self Press 1 - WORKING
Step 3: Check database for customer number - NOT WORKING

@dlpgh: Is this an old feature or is this an error occuring? Depending on your answer I'll know whether to get Ahmed working on it.

**Comment by dlpgh** [comment]
It's suppose to work!

Best,
Delaphone Gh Ltd.
Duke Ofori.
0242110110

sent from my android device

**Comment by jemymaRC** [comment]
@tektrix1: Please ensure that the system checks the database for the users telephone number after step 1 (To make a deposit for self press 1) is selected. Currently, the system rather asks the user to select their network provider after step 2 (Provide the amount you want to deposit).

**Comment by jemymaRC** [comment]
@tektrix1: After testing the "Make a deposit for self" option, the line cuts after an amount entered is confirmed by clicking 1. Please check this and fix.

**Comment by jemymaRC** [comment]
@tektrix1: After confirming an amount to deposit for self, the system requests that the user chooses a network. However, even when the correct network option is selected the system responds that it is incorrect and requests that the user call back later and cuts the line. Please check this.

Point of correction from an earlier comment I gave. As the user could be registered in the system with one number but be calling with another number, they should be able to use the system without an issue. However, this is a whole feature on its own which requires more thinking. So I'll ask that for now, check the system first to see if the number is registered, if not them redirect the user to customer service to register it. We will improve this step in future releases. (Please go to issue #5 for full instructions).

Apart from this issue, the script is incorrect. After the user enters an amount and confirms by pressing 1, they should be asked for their PIN:

b. Please enter the amount to deposit on your key pad and press # to confirm
Press# :
IVR: the amount to deposit is XX.XX Ghana Cedis: 
IVR: Press 1 to accept or press 2 to enter amount again:
i. Press 1 to accept amount
1. if amount OK: GOTO c.
ii. Press 2 to enter amount again
1. if amount OK: GOTO c.
2. If amount NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent

c. Please enter your pin and # to confirm
i. If pin OK: IVR: your request is now being processed. Thank You: GOTO c.
ii. If pin NOT OK: that is the wrong pin please try again
iii. If pin NOT OK after 3 attempts: sorry you having problems with your pin please wait while we connect you to our call center agent.
d. Please press 1 to return to the main menu else hang up. Thank you (This is now issue #6).

I am creating a new issue for each of these errors so we can keep track of your corrections and not get lost in too much feedback at once. I have added the issue numbers to each error so you can simply click on it to go to the issue entry and work on only one at a time.

**Comment by jemymaRC** [comment]
Tes results for 2. Make a deposit for a third party

Step 1: To Make a deposit for a third party press 2 - WORKING
Step 2: Please enter the customer’s mobile number and press # to confirm
Press# - WORKING
IVR response: The number you entered is XXXXXXXXXX - NOT WORKING

@tektrix1: The IVR is rather saying "The amount you entered is ...".

Please ensure that the response is as follows (also found in issue #8):

IVR: the number you entered is XXXXXXXXXX: 
IVR: Press 1 to accept or press 2 to enter number again:
i. Press 1 to accept number:
1. If number OK: GOTO b.
2. If number NOT OK: the number does not exist please try again (Number not in DB)
3. If number NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent

ii. Press 2 to enter number again:
1. If number OK: GOTO b.
2. If number NOT OK: the number does not exist please try again
3. If number NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent

b. Please enter the amount to deposit on your key pad and press # to confirm
Press# :
IVR: the amount to deposit is XX.XX Ghana Cedis: 
IVR: Press 1 to accept or press 2 to enter amount again:
i. Press 1 to accept amount
1. if amount OK: GOTO b.
ii. Press 2 to enter amount again
1. if amount OK: GOTO b.
2. If amount NOT OK after 3 attempts: sorry you having problems please wait while we connect you to our call center agent
b. Please enter your pin and press # to confirm
i. If pin OK: your request is now being processed. Thank You.: GOTO d.
ii. If pin NOT OK: that is the wrong pin please try again
iii. If pin NOT OK after 3 attempts: sorry you having problems with your pin please wait while we connect you to our call center agent.
c. Please press 1 to return to the main menu else hang up. Thank you.

**Comment by franklinodoi** [comment]
Passed test. Except for the stage where the Telco needs to be confirmed. It always reports that the number is invalid.
